src/cascadia/TerminalSettingsEditor/ColorSchemeViewModel.h

```diff
@@ -42,6 +49,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     private:
         winrt::hstring _Name;
         Model::ColorScheme _scheme;
+        Editor::ColorSchemesPageViewModel _parentPageVM{ nullptr };

```

Comment:
Concern: this could result in a memory leak if CSPVM has strong references to its CSVMs.

Reply:
made this a weak ref to the parent page (and do the `.get` thing when we're about to use it)